lib/util/git/index.ts

```diff
@@ -585,7 +585,7 @@ export async function getFileList(): Promise<string[]> {
 }
 
 export function getBranchList(): string[] {
-  return Object.keys(config.branchCommits);
+  return Object.keys(config.branchCommits ?? /* istanbul ignore next */ {});
```

Comment:
I discovered this bug while using an invalid forkToken, it threw a fatal error so fixing it here